site/src/pages/[shortname]/[blockSlug].page.tsx

```diff
@@ -114,35 +112,20 @@ export const getStaticProps: GetStaticProps<
     return { notFound: true };
   }
 
+  const packagePath = `${shortname}/${blockSlug}`;
   const catalog = readBlocksFromDisk();
 
-  const blockMetadataResponse = await fetch(
-    `${BASE_URL}/blocks/${shortname}/${blockSlug}/block-metadata.json`,
+  const blockMetadata = catalog.find(
+    (metadata) => metadata.packagePath === packagePath,
   );
```

Comment:
This is more expensive than it should be, as we are reading all the blocks from the disk just to find a specific one. Ideally we'd have a `readBlockFromDisk` function to use instead. I appreciate this is likely out of scope for this PR, so if you could leave a `@todo` here that would be great.

Reply:
I agree. Will update

Reply:
Added a @todo for this here acef6d4